note/15. Match.md

```
@@ -0,0 +1,77 @@
+# Match
+## The Python Match Statement
+Instead of writing many if..else statements, you can use the match statement.
+
+The match statement selects one of many code blocks to be executed.
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)